src/js/modules/articles/components/ArticlePage.js

```diff
@@ -12,18 +15,74 @@ import {
 
 const styles = {
   ArticlePage: {
-    margin: "0 auto",
-    maxWidth: "1060px",
+    marginTop: "60px",
+  },
+  descriptionRow: {
+    marginBottom: "24px",
+  },
+  description: {
+    border: "1px solid #ddd",
+    borderStyle: "solid none", // only top-bottom borders
+    color: "#000",
+    fontFamily: "Minion Pro",
+    fontSize: "16px",
+    padding: "12px 0 13px",
+  },
+  subscribe: {
+    color: "#3572b7",
+    "&:active": {
+      color: "#3572b7",
+    },
+    "&:focus": {
+      color: "#3572b7",
+    },
+    "&:hover": {
+      color: "#3572b7",
+    },
+  },
+  "@media (max-width: 1199px)": {
+    ArticlePage: {
+      padding: "0 8%",
+    },
+  },
+  "@media (max-width: 991px)": {
+    ArticlePage: {
+      padding: 0,
+    },
+    descriptionRow: {
+      padding: "0 10%",
+    },
+  },
+  "@media (max-width: 767px)": {
+    descriptionRow: {
+      padding: "0 2%",
+    },
   },
 };
 
+//TODO: pop it up
 const ArticlePage = ({ classes, article, section, featuredMedia }) => {
   return (
-    <div className={classes.ArticlePage}>
+    <Grid fluid className={classes.ArticlePage}>
       <ArticleHeader article={article} section={section} />
       <ArticleBody content={article.content} featuredMedia={featuredMedia} />
+      <Row className={classes.descriptionRow}>
+        <Col xs={12} sm={12} md={9} lg={9} className={classes.description}>
+          The Pulse of the Student Body:&nbsp;
+          <Link
+            to={"/maybe-we-should-pop-up-the-subscribe-modal"}
```

Comment:
This is fine now, but maybe make a TODO to remove it. Wouldn't want to have it make it into production